doc/content/template.md

```diff
@@ -0,0 +1,139 @@
+!table id=fuel_salt_properties caption=Thermophysical properties of the fuel salt.
+|   |   | Unit  | LiF-BeF$_4$-ZrF$_4$-UF$_4$  |
+| :- | :- | :- | :- |
+| Melting temperature | $T_{melt}$ | $K$ | $722.15$  |
+| Density | $\rho$ | $kg/m^3$  | $2553.3-0.562\bullet T$ |
+| Dynamic viscosity | $\mu$ | $Pa\bullet s$ | $8.4\times 10^{-5} exp(4340/T)$ |
+| Thermal conductivity | $k$ | $W/(m\bullet K)$ | $1.0$ |
+| Specific heat capacity | $c_p$ | $J/(kg\bullet K)$ | $2009.66$ |
+
+```
+
+[Click here](msr/msre/msre_sam_model.md#fuel_salt_properties) to check out the formatted table appeared on VTB website:
+
+#### Adding references
+
+If the model has been previously published, one should provide the reference to the published works together with the key references used for model development. 
+The Latex style references are supported in MOOSE documentation, and some examples about how to cite technical report, conference proceedings and journal articles are given below. The bibtex entries have to be added into file `vtb.bib`. 
+A bibliography list will be generated automatically at the end of the document. 
+
+```language=bash
+@techreport{Hu2017,
+        address = {Lemont, IL},
+        author = {Hu, Rui},
+        number = {ANL/NE-17/4},
+        mendeley-tags = {ANL/NE-17/4},
+        institution = {Argonne National Laboratory},
+        title = {{SAM Theory Manual}},
+        year = {2017}
+}
+
+```
+
+```language=bash
+@InProceedings{novak_2021c,
+  title       = {{Coupled {Monte} {Carlo} and Thermal-Hydraulics Modeling of a Prismatic Gas Reactor Fuel Assembly Using {Cardinal}}},
+  author     = {A.J. Novak and D. Andrs and P. Shriwise and D. Shaver and P.K. Romano and E. Merzari and P. Keutelian},
+  booktitle  = {{Proceedings of Physor}},
+  year       = 2021
+}
+
+```
+
+```language=bash
+@article{Fang2021,
+        author = {Fang, Jun and Shaver, Dillon R. and Min, Misun and Fischer, Paul
+                and Lan, Yu-Hsiang and Rahaman, Ronald and Romano, Paul
+                and Benhamadouche, Sofiane and Hassan, Yassin A.
+                and Kraus, Adam and Merzari, Elia},
+        doi = {10.1016/j.nucengdes.2021.111143},
+        journal = {Nuclear Engineering and Design},
+        title = {{Feasibility of Full-Core Pin Resolved CFD Simulations of
+                Small Modular Reactor with Momentum Sources}},
+        volume = {378},
+        year = {2021}
+}
+
+```
+
+To cite the references, one can do the following in the markdown file.
+
+```language=bash
+description part A [!citep](Hu2017), description part B [!citep](novak_2021c), description part C [!citep](Fang2021)
+
+```
+
+description part A [!citep](Hu2017), description part B [!citep](novak_2021c), description part C [!citep](Fang2021)
+
+
+#### Other key model details to provide in the high-level summary
+
+- MOOSE Based Applications Used: Griffin, Heat Conduction, Cardinal, etc. (Please click here to see the checklist, this can be used for Guillaume to categorize/sort)
```

Comment:
```suggestion
- MOOSE Based Applications Used: Griffin, Heat Conduction, Cardinal, etc. (Please click here to see the checklist, this can be used for the VTB team to categorize/sort the model)
```